crates/mun_runtime/src/lib.rs

```diff
@@ -22,6 +22,31 @@ pub struct RuntimeOptions {
     pub delay: Duration,
 }
 
+/// A builder for the runtime.
+pub struct RuntimeBuilder {
+    options: RuntimeOptions,
```

Comment:
Are these options then still required? Feels strange to have an intermediate object (`RuntimeOptions`).

Reply:
Either I need to maintain separate variables for all of the things that will inevitably end up in `RuntimeOptions`, or do it this way. I felt that this was cleaner.